### DISCUSSION TITLE
How to automatically rename files after moving and strip characters?

### DISCUSSION THREAD

**Discussion by fliptoback** [tfeldmann/organize#244] [Q&A]
Hi, I have many files which are named abc (xyz.org).pdf for instance. Whilst I can move all of these files to a PDF folder, I am struggling how to strip the (xyz.org) from the original filename.

Could someone point me in the right direction?

Thanks in advance.

**Comment by telometto** [comment]
Hi!

Can you post your config? You can strip whatever you'd like from the file name by using regex. It'd be something along the lines of
`{name.replace('(xyz.org)', '')}`. But please post your config before using the regex 😊
